include/readout/ReadoutTypes.hpp

```diff
@@ -55,9 +57,9 @@ struct WIB_SUPERCHUNK_STRUCT
   // comparable based on first timestamp
   bool operator<(const WIB_SUPERCHUNK_STRUCT& other) const
   {
-    auto thisptr = reinterpret_cast<const dunedaq::dataformats::WIBHeader*>(&data);        // NOLINT
-    auto otherptr = reinterpret_cast<const dunedaq::dataformats::WIBHeader*>(&other.data); // NOLINT
-    return thisptr->get_timestamp() < otherptr->get_timestamp() ? true : false;
+    // auto thisptr = reinterpret_cast<const dunedaq::dataformats::WIBHeader*>(&data);        // NOLINT
+    // auto otherptr = reinterpret_cast<const dunedaq::dataformats::WIBHeader*>(&other.data); // NOLINT
+    return this->get_timestamp() < other.get_timestamp();
```

Comment:
Aggregation concepts can be misleading and cause confusion until we don't move any "custom type" specific header and utility to an FE package. For that, I would propose to change the get_timestamp interface to:
`get_timestamp(unsigned idx)` that checks that idx is within aggregation range. Make idx mandatory, don't add idx=0 default in the signature.